src/manifold/src/manifold.cpp

```diff
@@ -919,12 +919,12 @@ Manifold Manifold::Hull(const std::vector<glm::vec3>& pts) {
   const int numVert = pts.size();
   if (numVert < 4) return Manifold();
 
-  std::vector<quickhull::Vector3<double>> vertices(numVert);
+  std::vector<glm::vec3> vertices(numVert);
   for (int i = 0; i < numVert; i++) {
     vertices[i] = {pts[i].x, pts[i].y, pts[i].z};
   }
 
-  quickhull::QuickHull<double> qh;
+  quickhull::QuickHull<float> qh;
```

Comment:
why float?

Reply:
So, glm::vec3 has floats right?, so since some of the operations that were originally a part of Vecto3.hpp would give float outputs, and others double and then typecasting would happen, I thought instead let's just use floats, especially since the output we store is in floats too, if you want I can change that though.

Reply:
just use `glm::dvec3`, we will soon switch to double.

Reply:
Alright, I've shifted it to glm::dvec3 instead.